examples/digital_fingerprinting/production/morpheus/dfp/utils/model_cache.py

```diff
@@ -331,11 +331,10 @@ def load_model_cache(self, client: MlflowClient, reg_model_name: str, timeout: f
             logger.error("Deadlock when trying to acquire model cache lock")
             raise RuntimeError("Deadlock when trying to acquire model cache lock") from e
 
-    # pylint: disable=dangerous-default-value
     def load_user_model_cache(self,
                               user_id: str,
                               timeout: float,
-                              fallback_user_ids: typing.List[str] = []) -> UserModelMap:
+                              fallback_user_ids: typing.List[str] = None) -> UserModelMap:
```

Comment:
I assume that `None` is checked later on in `UserModelMap`? Can you confirm?

Reply:
Check is not performed at `UserModelMap`. I've relocated it to `ModelManager.load_user_model`, which is the starting point for loading the model.